examples/hotel-bookings/src/index.md

```diff
@@ -0,0 +1,335 @@
+---
+toc: false
+theme: [air, ocean-floor, wide]
+---
+
+# Hotel reservations by market segment
+
+## Excludes complementary reservations
+
+```js
+import {DonutChart} from "./components/donutChart.js";
+import {bigNumber} from "./components/bigNumber.js";
+
+const hotelData = FileAttachment("data/hotelData.csv").csv({typed: true})
+```
+
+```js
+// Radio button input to choose market segment
+const pickMarketSegmentInput =
+  Inputs.radio(
+    ["All"].concat(hotelData
+      .filter((d) => d.MarketSegment != "Complementary")
+      .map((d) => d.MarketSegment)),
+    {
+      label: "Booking type:",
+      value: "All",
+      unique: true
+    }
+  )
+;
+
+const pickMarketSegment = Generators.input(pickMarketSegmentInput);
+```
+
+${pickMarketSegmentInput}
+
+## ${pickMarketSegment} bookings
+
+<div class="grid grid-cols-4">
+  <div class="card grid-rowspan-2">
+    ${resize(width => DonutChart(byCountry, {centerText: "Guest nationality", width: width, colorScale: d3.quantize(t => d3.interpolateTurbo(t * 0.2 + 0.02), 6)}))}
```

Comment:
the color matching between countries and colors should be consistent across market segments

Reply:
Done (using Observable10 palette)